model-optimizer/extensions/front/kaldi/split_lstm_memoryoffset.py

```diff
@@ -0,0 +1,58 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import networkx as nx
+
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph
+from mo.ops.memoryoffset import MemoryOffset
+from mo.ops.result import Result
+from mo.utils.error import Error
+
+
+class SplitLstmMemoryOffset(FrontReplacementSubgraph):
+    """
+    Splits MemoryOfffsets in LSTM blocks to avoid cycles during shape inference.
```

Comment:
Extra `f`

Reply:
Removed extra f and added more informative description of this transformation.